src/mock/MockData.cpp

```diff
@@ -296,31 +297,77 @@ std::unordered_map<std::string, std::vector<Serve>> MockData::teamServes_ = team
 // Mock schema
 std::shared_ptr<meta::NebulaSchemaProvider> MockData::mockPlayerTagSchema(SchemaVer ver) {
     std::shared_ptr<meta::NebulaSchemaProvider> schema(new meta::NebulaSchemaProvider(ver));
-    schema->addField("name", meta::cpp2::PropertyType::STRING, 0, false, "");
+    schema->addField("name",
```

Comment:
Maybe we can provide a overload function `addField` and construct the constant expression inside, use the proxy pattern like following:

```c++
void addField(xx, ..., value) {
  addField(xx,..., new ConstantExpression(value));
}
```

So you needn't to update so many cases.

Reply:
I don't think that's necessary, because normally it wouldn't be a value, it would be an expression.